Composer/packages/ui-plugins/lg/src/LgField.tsx

```diff
@@ -36,10 +36,21 @@ const getInitialTemplate = (fieldName: string, formData?: string): string => {
 
 const LgField: React.FC<FieldProps<string>> = props => {
   const { label, id, description, value, name, uiOptions } = props;
-  const { designerId, currentDialog, lgFiles, shellApi, projectId, locale, userSettings } = useShellApi();
-
+  const { designerId, currentDialog, lgFiles, shellApi, projectId, locale, userSettings, data } = useShellApi();
+
+  let lgType = name;
+  const kind = data?.$kind;
+  if (kind) {
+    const [, schemaType] = kind.split('.');
+    if (schemaType === 'SendActivity') {
+      lgType = schemaType;
+    } else {
+      const nameCapitalized = name.charAt(0).toUpperCase() + name.slice(1);
+      lgType = `${schemaType}_${nameCapitalized}`;
```

Comment:
Considering the 'schema-driven' pattern, we need to consume $kind string from `SDKKinds` instead of harcoding the `SendActivity` string.
```ts
if (kind === SDKKinds.SendActivity) {
  ...
}
```

Reply:
I'm also curious about why only `SendActivty` has a special naming pattern instead of naming it to `'SendActivity_activity_xxxx'`?

Reply:
Thanks remind! 